src/index.ts

```diff
@@ -707,7 +708,7 @@ or make sure that you already have Serverless 3.x installed in your project.
   }
 
   private getHandlerWrapper(runtime: string, handler: string) {
-    if (["nodejs12.x", "nodejs14.x"].indexOf(runtime) !== -1) {
+    if (runtime.startsWith("nodejs")) {
```

Comment:
I'd prefer that we do `["nodejs12.x", "nodejs14.x","nodejs16.x"].indexOf(runtime) !== -1` instead of screening solely on the node prefix.

Reply:
Thanks for your review @mrickard ! 😊 

Since I want love this to be released quickly, I've done it, however: 
- this forces us to change the code in two locations when we have a new Node.js runtime
- we still have some other kind of logic for determining the python wrapper (using pattern matching instead of exact matching).

Why not align the logic for all three wrappers like the following?

```ts
const nodeJsRuntimes = [
  "nodejs12.x",
  "nodejs14.x",
  "nodejs16.x",
];

const pythonRuntimes = [
  "python3.6",
  "python3.7",
  "python3.8",
  "python3.9",
];

const javaRuntimes = [
  "java11",
  "java8.al2",
];

const wrappableRuntimeList = [
  ...nodeJsRuntimes,
  ...pythonRuntimes,
  ...javaRuntimes,
];

// ...

  private getHandlerWrapper(runtime: string, handler: string) {
    if (nodeJsRuntimes.indexOf(runtime) !== -1) {
      return "newrelic-lambda-wrapper.handler";
    }

    if (pythonRuntimes.indexOf(runtime) !== -1) {
      return "newrelic_lambda_wrapper.handler";
    }

    if (javaRuntimes.indexOf(runtime) !== -1) {
      return `com.newrelic.java.HandlerWrapper::${this.javaNewRelicHandler}`;
    }

    return handler;
  }
```

Happy to create an issue to talk about these problems and to create another PR to do the above.

That way it'll be easier for others to understand what we're doing here.